base_export_manager/models/ir_exports_line.py

```diff
@@ -0,0 +1,222 @@
+                # Translate label if possible
+                try:
+                    parts.append(
+                        one.env[one.model_n(num).model]
+                        ._fields[field.name]
+                        .get_description(one.env)["string"]
+                    )
+                except KeyError:
+                    # No human-readable string available, so empty this
+                    return
+            one.label = (
+                "{} ({})".format("/".join(parts), one.name)
+                if parts and one.name
+                else False
+            )
+
+    def _inverse_name(self):
+        """Get the fields from the name."""
+        for one in self:
+            # Field names can have up to only 4 indentation levels
+            parts = one.name.split("/")
+            if len(parts) > 4:
+                raise exceptions.ValidationError(
+                    _("It's not allowed to have more than 4 levels depth: %s")
+                    % one.name
+                )
+            for num in range(1, 5):
+                if not any(parts) or num > len(parts):
+                    # Empty subfield in this case
+                    # You could get to failing constraint while populating the
+                    # fields, so we skip the uniqueness check and manually
+                    # check the full constraint after the loop
+                    one.with_context(skip_check=True)[one.field_n(num, True)] = False
+                    continue
+                field_name = parts[num - 1]
+                model = one.model_n(num)
+                # You could get to failing constraint while populating the
+                # fields, so we skip the uniqueness check and manually check
+                # the full constraint after the loop
+                one.with_context(skip_check=True)[
+                    one.field_n(num, True)
+                ] = one._get_field_id(model, field_name)
+            # invalidate_cache -> in order to get actual value of field 'label'
+            # in function '_check_name'
+            self.invalidate_cache(ids=one.ids)
+            one._check_name()
+
+    @api.constrains("field1_id", "field2_id", "field3_id", "field4_id")
+    def _check_name(self):
+        # do also skip the check if label is set or not, when skip_check is set
+        if not self._context.get("skip_check"):
+            for one in self:
+                if not one.label:
+                    raise exceptions.ValidationError(
+                        _("Field '%s' does not exist") % one.name
+                    )
+                lines = one.search(
+                    [("export_id", "=", one.export_id.id), ("name", "=", one.name)]
+                )
+                if len(lines) > 1:
+                    raise exceptions.ValidationError(
+                        _("Field '%s' already exists") % one.name
+                    )
+
+    @api.onchange("name")
+    def _onchange_name(self):
```

Comment:
I think you can remove this method and add api.depends("name") to _inverse_name method